MatrixSDK/JSONModels/Authentication/MXAuthMetadata.m

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright 2025 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "MXAuthMetadata.h"
+
+static NSString *const kMXIssuer = @"issuer";
+static NSString *const kMXAccountManagementUri = @"account_management_uri";
+static NSString *const kMXAccountManagementActionsSupported = @"account_management_actions_supported";
```

Comment:
Not super important seeing as they're private, but generally these would be slightly more descriptive, something along the lines of

`kMXAuthMetadataIssuer`/`kMXAuthMetadataTypeIssuer`/`kMXIssuerJSONKey`